platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -27,7 +27,6 @@
 import java.util.function.Consumer;
 import java.util.stream.Stream;
 import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.Validate;
 import org.apache.karaf.system.SystemService;
 import org.codice.ddf.migration.Migratable;
```

Comment:
❗️ Make sure to delete the key file when export failure occurs and when import succeeds just as we delete the zip.

Reply:
yep, will add this now